propiedadesCR/src/main/java/com/techflow/propiedadesCR/services/PermissionsService.java

```diff
@@ -0,0 +1,46 @@
+package com.techflow.propiedadesCR.services;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.techflow.propiedadesCR.contracts.PermissionsRequest;
+import com.techflow.propiedadesCR.ejb.Tpermission;
+import com.techflow.propiedadesCR.pojo.PermissionPOJO;
+import com.techflow.propiedadesCR.repositories.PermissionsRepository;
+/**
+ * <h1>Servicio del permiso</h1>
+ * Clase encargada de ofrecer los servicios del permiso
+ * y administrar las transacciones al repositorio
+ *@author Valeria Ramírez
+ *@version 1.0
+ *@since 03/03/2016
+ */
+@Service
+public class PermissionsService implements PermissionsServiceInterface {
+	/*
+	 * Objeto que se comunica con la base de datos
+	 */
+	@Autowired private PermissionsRepository permissionsRepository;
+	
+	@Override
+	@Transactional
+	/**
+	 * Método que levanta todos los permisos del sistema
+	 * @return generateRoleData lista que contiene todos los roles del sistema
```

Comment:
Buen trabajo Jimmi!

> On Mar 4, 2016, at 11:58 AM, jimmivco <notifications@github.com> wrote:
> 
> In propiedadesCR/src/main/java/com/techflow/propiedadesCR/services/PermissionsService.java <https://github.com/techflowCenfotec/propiedadesCR/pull/32#discussion_r55063970>:
> 
> > + *@author Valeria Ramírez
> > + *@version 1.0
> > + *@since 03/03/2016
> > + */
> > +@Service
> > +public class PermissionsService implements PermissionsServiceInterface {
> > +	/*
> > +	 * Objeto que se comunica con la base de datos
> > +	 */
> > +	@Autowired private PermissionsRepository permissionsRepository;
> > +	
> > +	@Override
> > +	@Transactional
> > +	/**
> > +	 * Método que levanta todos los permisos del sistema
> > +	 * @return generateRoleData lista que contiene todos los roles del sistema
> mayuscula despues de generateRoleData
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/techflowCenfotec/propiedadesCR/pull/32/files#r55063970>.
>